android/src/main/java/com/stripeterminalreactnative/listener/RNReaderReconnectionListener.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripeterminalreactnative.listener
```

Comment:
can we add tests for this? example: https://github.com/stripe/stripe-terminal-react-native/blob/1870117c/android/src/test/java/com/stripeterminalreactnative/listener/RNBluetoothReaderListenerTest.kt#L1